Components/Molecules/GoalItem.js

```diff
@@ -35,6 +36,24 @@ const lastTimeStampMessageCreate = lastTimeStamp => {
   return moment(lastTimeStamp).format("M/DD(ddd) h:mma"); //fromNow();
 };
 
+const GoalSubTitleTextBasic = ({ totalCount, cadenceCount, cadence }) => {
+  return `${totalCount}/${cadenceCount} (${cadence})`;
+};
+
+const GoalSubTitleTextStreak = ({ streakCount, totalCount, cadenceCount, cadence }) => {
+  const cadenceText = cadence == "daily" ? "Day" : cadence == "weekly" ? "Wk" : "Mo";
```

Comment:
My daily goals are showing **Mo** 

As I slacked you, its because we are saving it **daily** from somewhere and **Daily** from another. In result this ternary is defaulting to Mo string dont match. 

Currently in the DB we have: **Monthly, Weekly, Daily, and daily**. I will update daily --> Daily with a query

Reply:
On second look, it looks like the **cadence** is coming through as **Daily**. And since its not matching **daily**, its defaulting to Mo.